### PR TITLE
SONAR-6576 WS api/metrics/domains list all the available metric domains available

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

File: server/sonar-server/src/main/java/org/sonar/server/metric/ws/DomainsAction.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      json.close();
+    } finally {
+      MyBatis.closeQuietly(dbSession);
+    }
+  }
+
+  private void writeDomains(JsonWriter json, List<String> domains) {
+    json.name("domains");
+    json.beginArray();
+    for (String domain : domains) {
```
Comment (jblievremont) [inline server/sonar-server/src/main/java/org/sonar/server/metric/ws/DomainsAction.java]: You could have used ``json.values(domains);`` instead.

**Comment by jblievremont** [comment]
Very small feedback, and LGTM :+1:

File: server/sonar-server/src/main/java/org/sonar/server/metric/ws/DomainsAction.java (hunk 1 above)
Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/metric/ws/DomainsAction.java]: thanks, I didn't know about it

**Pull request merged**